src/templates/common/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  transformBreadcrumbs,
+  deriveLastBreadcrumbFromPath,
+  deriveLcBreadcrumbs,
+  filterInvalidCrumbs,
+} from '@/templates/globals/util/breadcrumbUtils'
+import { BreadcrumbItem } from '@/types/dataTypes/drupal/field_type'
+
+interface BreadcrumbProps {
+  breadcrumbs?: BreadcrumbItem[]
+  breadcrumbTitle?: string
+  disableAnalytics?: boolean
+  deriveBreadcrumbsFromUrl?: boolean
+  replaceLastItem?: boolean
+  constructLcBreadcrumbs?: boolean
+  titleInclude?: boolean
+  hideHomeBreadcrumb?: boolean
+  customCrumbHomeText?: string
+  entityPath?: string
+}
+const Breadcrumbs = ({
+  breadcrumbs,
+  entityPath,
+  breadcrumbTitle,
+  disableAnalytics,
+  deriveBreadcrumbsFromUrl,
+  replaceLastItem,
+  constructLcBreadcrumbs,
+  titleInclude,
```

Comment:
I feel like this prop could maybe be more clear if it included `lc` somehow (e.g. `lcBreadcrumbsTitleInclude`). As I understand it, this prop is only useful when `constructLcBreadcrumbs` is `true`, but it _feels_ like it's named in a way that might make me (as a consumer) feel like I can set it in all cases.

Reply:
agreed, updated